File: 0242-valid-anagram/0242-valid-anagram.cpp
class Solution {
public:
    bool isAnagram(string s, string t) {
        unordered_map<char,int>s1;
        unordered_map<char,int>t1;
        if(s.size()!=t.size())return false;
        for(char c:s){
            s1[c]++;
        }
        for(char c:t){
            t1[c]++;
        }
        for(auto it:s1){
            if(t1[it.first]!=it.second){
                return false;
            }
        }
        return true;
    }
};

File: 0242-valid-anagram/0242-valid-anagram.py
class Solution(object):
    def isAnagram(self, s, t):
        if(len(s)!=len(t)):
            return False
        s1=[0]*128
        t1=[0]*128
        for i in range(len(s)):
            s1[ord(s[i])]+=1
            t1[ord(t[i])]+=1
        return s1==t1